zebra-state/src/service.rs

```diff
@@ -540,12 +535,11 @@ impl StateService {
 
     /// Send an error on a `QueuedFinalized` block's result channel, and drop the block
     fn send_finalized_block_error(queued: QueuedFinalized, error: impl Into<BoxError>) {
-        let (finalized, rsp_tx) = queued;
+        let (_finalized, rsp_tx) = queued;
 
         // The block sender might have already given up on this block,
         // so ignore any channel send errors.
         let _ = rsp_tx.send(Err(error.into()));
-        std::mem::drop(finalized);
```

Comment:
Nitpick: I'm not sure why this got changed, it seems more explicit the other way?

Reply:
I just noticed that it was already happening implicitly, but it's unrelated to the PR so I'll put it back.